components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/DefaultIDTokenBuilder.java

```diff
@@ -165,6 +166,22 @@ public String buildIDToken(OAuthTokenReqMessageContext tokenReqMsgCtxt,
             jwtClaimsSet.setClaim(OAuthConstants.OIDCClaims.ACR, "urn:mace:incommon:iap:silver");
         }
 
+        List<ClaimProvider> claimProviders = OAuth2ServiceComponentHolder.getClaimProviders();
+        if (claimProviders != null) {
```

Comment:
Can this be null?

Reply:
Also can we move this to a private method

Reply:
If there is no any claimProviders, then can't it be null?

Reply:
We initialize an empty list and add claim providers. We can't directly access the list of claim providers so there is no chance of someone assiging NULL right?

Reply:
+1